utility-emissions-channel/typescript_app/tests/test.js

```diff
@@ -160,6 +160,40 @@ describe("Test fabric", function() {
       });
   });
 
+
+ it("should check if partyId is Encrypted", function(done) {
+    chai
+       .request(host)
+      .post(recordEmissionPath)
+      .set("content-type", "application/x-www-form-urlencoded")
+      .send({
+        userId: testUser,
+        orgName: "auditor1",
+        utilityId: "USA_EIA_11208",
+        partyId: "1234567890",
+        fromDate: "2020-04-05T10:10:09Z",
+        thruDate: "2020-04-05T10:10:09Z",
+        energyUseAmount: 100,
+        energyUseUom: "kWh",
+      })
+       .end((error, response) => {
+         
+         try{
+           let entry = response.body;
+           expect(entry.info).to.equal("EMISSION RECORDED TO LEDGER");
+           expect(entry.partyId).to.not.equal("1234567890");
+           
+           // checking if partyId is encrypted
+           expect(entry.partyId).to.equal(hash);  
```

Comment:
Hi @udosson . Thanks for your response.
This line solves the main part for testing, if partyId is encrypted.

Please let me know, if it needs changes. Thanks!

Reply:
As @petermetz mention, the function tests if the partyId is hashed. Could you update your comment in line 186, please?

Reply:
Done! Thanks for the suggestion!